go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/SSHcom/privx-sdk-go
 
-go 1.18
+go 1.19
```

Comment:
While we are already upgrading the go version, should we upgrade it to the latest version 1.20? We could also upgrade the deps and update the go.mod file.

Reply:
yea, i guess we can.
I didnt upgrade to 1.19 because I took reference from our main privx repo, and seemed to me like it is in 1.19, so i only went up to now.
I guess going to 1.20 shouldnt be a problem.
And yes, the deps could also be updated/

Reply:
done